topics/metagenomics/tutorials/beer-data-analysis/tutorial.md

```diff
@@ -0,0 +1,391 @@
+
+![FastQC Per base sequence quality with scores below 200](./images/fastqc_1.png "Per base sequence quality")
+
+We can see that the quality of our sequencing data grows after the first few bases, stays around a score of 18, which is a relatively low value compared to other sequencing technologies, and then decreases again at the end of the sequences.
+
+## Improve the dataset quality
+
+In order to improve the quality of our data, we will use two tools:
+- **porechop** ({% cite rrwick2017 %}) to remove adapters that were added for sequencing and chimera (contaminant)
+- **fastp** ({% cite Chen_2018 %}) to filter sequences with low quality score (below 9)
+
+> ### {% icon hands_on %} Hands-on: Improve the dataset quality
+>
+> 1. {% tool [Porechop](toolshed.g2.bx.psu.edu/repos/iuc/porechop/porechop/0.2.3) %} with the following parameters:
+>    - *"Input FASTA/FASTQ"*: {% icon param-files %} `Reads`
+>    - *"Output format for the reads"*: `fastq`
+>
+> 2. {% tool [fastp](toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.23.2+galaxy0) %} with the following parameters:
+>    - *"Single-end or paired reads"*: `Single-end`
+>     - {% icon param-collection %} *"Dataset collection"*: output of **Porechop**
+>        - In *"Adapter Trimming Options"*:
+>            - *"Disable adapter trimming"*: `Yes`
+>    - In *"Filter Options"*:
+>        - In *"Quality filtering options"*:
+>            - *"Qualified quality phred"*: `9`
+>    - In *"Read Modification Options"*:
+>        - *"PolyG tail trimming"*: `Disable polyG tail trimming`
+>
+> 3. (Optional) {% tool [FASTQC](toolshed.g2.bx.psu.edu/repos/devteam/fastqc/fastqc/0.73+galaxy0) %} with the following parameters
+>    - {% icon param-files %} *"Raw read data from your current history"*: `output of **fastp**
+>
+> 4. (Optional) Inspect the output of **FASTQC** to see how the quality has been improved
+{: .hands_on}
+
+# Assign taxonomic classifications
+
+One of the key steps in metagenomic data analysis is to identify the taxon to which the individual reads belong.
+
+> ### {% icon comment %} Taxon?
+>
+> Taxonomy is the method used to naming, defining (circumscribing) and classifying groups of biological organisms based on shared characteristics. It is founded on the concept that morphological similarities descend from a common evolutionary ancestor.
+>
+> At a very broad level, all living organisms are grouped into one of the 7 Kingdoms of Life each of which are split further into Phylum > Class > Order > Family > Genus > Species.
+>
+> For example for the cat:
+>
+> Level | Classification
+> --- | ---
+> Kingdom | Animalia
+> Phylum | Chordata
+> Class | Mammalia
+> Order | Carnivora
+> Family | Felidae
+> Genus | *Felis*
+> Species |	*F. catus*
+>
+> A taxon is a then group of organisms or of population(s) of organisms that form a taxonomic unit. A taxon is usually assigned a rank when it is given its formal, Latin name.
```

Comment:
Somthing is wrong here I think:
> A taxon is a then a group of organisms or of population(s) of organisms that form a taxonomic unit. 

Maybe:
A taxon is a group of organisms and/ or populations(s) of organisms that together form a taxonomic unit.